homework/web_scraping_sample.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup 
+import psycopg2 
+
+
+#make a request to the website
+url='https://blog.python.org/'
+response=requests.get(url)
+
+#parse the html content using BeautifulSoup
+soup=BeautifulSoup(response.content,'html.parser')
+
+#find all article titles and links
+titles=soup.find_all('h3',class_='blog-title')
+dates=soup.find_all('h2',class_='date-header')
+	
+#connect to the PostgreSQL database
+conn=psycopg2.connect(database="mydatabase",user="postgres",password="123456",host="localhost",port="5432")
```

Comment:
@sampleharvgithub Have you tested this script from your end?
If yes, did you verify that the data is getting saved in postgresql database that you've used?